src/Microsoft.Identity.Client/ApiConfig/SystemWebViewOptions.cs

```diff
@@ -68,5 +73,85 @@ internal void LogParameters(ICoreLogger logger)
             logger.InfoPii("BrowserRedirectError " + BrowserRedirectError,
                "BrowserRedirectError? " + (BrowserRedirectError != null));
         }
+
+#if NET_CORE || DESKTOP || NETSTANDARD
+
+        /// <summary>
+        /// Use Microsoft Edge to navigate to the given uri. On non-windows platforms it uses 
+        /// whatever browser is the default
+        /// </summary>
+        /// <remarks>This helper is experimental and the signature may change without a major version increment.</remarks>
+        public static async Task OpenWithEdgeBrowserAsync(Uri uri)
+        {
+            if (uri == null)
+            {
+                throw new ArgumentNullException(nameof(uri));
+            }
+
+            string url = uri.AbsoluteUri;
+
+#if DESKTOP
+            url = url.Replace("&", "^&");
+            Process.Start(new ProcessStartInfo("cmd", $"/c start microsoft-edge:{url}") { CreateNoWindow = true });
+            await Task.FromResult(0).ConfigureAwait(false);
+#else
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Why are these details not encapsulated in the platform proxy?

Reply:
There are just some helpers that we can show to users if they need to see an implementation. Platform proxy has the logic of opening the url in the default browser.